scripts/linux/trcE2ETest.sh

```diff
@@ -296,18 +296,38 @@ function print_deployment_logs() {
 
 function get_support_bundle_logs(){
 
-    print_highlighted_message "Getting Support Bundle Logs"
+    print_highlighted_message "Getting Support Bundle Logs WITH TIMEOUT"
     mkdir -p $working_folder/support
     time=$(echo $test_start_time | sed 's/ /T/' | sed 's/$/Z/')
-    iotedge support-bundle -o $working_folder/support/iotedge_support_bundle.zip --since "$time"
-    print_highlighted_message "Finished getting support Bundle Logs"
+
+    MAX_RETRIES=4
+    RETRY_COUNT=0
+    DID_TIMEOUT=false
+
+    while [ "$RETRY_COUNT" -lt "$MAX_RETRIES" ] && [ "$DID_TIMEOUT" = false ]; do
```

Comment:
If I'm reading this logic correctly, we'll only retry once. If there's a timeout, then `DID_TIMEOUT` will be set to true and the while condition above will fail.

Reply:
Will fix rushed through this part.